johnzon-jsonb/src/main/java/org/apache/johnzon/jsonb/JohnzonBuilder.java

```diff
@@ -234,6 +236,10 @@ public Jsonb build() {
                         Integer.parseInt(it.toString()))
                 .ifPresent(builder::setSnippetMaxLength);
 
+        // drop johnzon-mapper BigInteger/BigDecimal built in adapters to not serialize those types as JsonString. See JSON-B spec 3.4.1
```

Comment:
1. maybe replace the stored list by two toggles since we don't aim at storing it in mem at runtime there (this is used in a fallback of fallback we can rework later ;))
2. for me string is the right choice by default (see https://github.com/jakartaee/jsonb-api/issues/187) so maybe introduce a toggle in the jsonb config (and fallback on a system prop - static -  to easily pass tck?)

:+1: for te test

Reply:
1. reverted all changes I made in LazyConverterMap and simply added two booleans to enable/disable BigDecimal/Integer string adapters
2. Now defaults to string, though I do have mixed feelings about this one. Defaulting to string absolutely makes sense, but we're actively ignoring json-b spec here now